yt_dlp/extractor/pialive.py

```diff
@@ -0,0 +1,86 @@
+from .common import InfoExtractor
+from .piaulizaportal import PIAULIZAPortalAPIIE
+from ..utils import ExtractorError, multipart_encode, smuggle_url
+
+
+class PiaLiveIE(InfoExtractor):
+    PLAYER_ROOT_URL = 'https://player.pia-live.jp'
+    PIA_LIVE_API_URL = 'https://api.pia-live.jp'
+    _VALID_URL = r'https?://player\.pia-live\.jp/stream/(?P<id>[\w-]+)'
+
+    _TESTS = [
+        {
+            'url': 'https://player.pia-live.jp/stream/4JagFBEIM14s_hK9aXHKf3k3F3bY5eoHFQxu68TC6krUDqGOwN4d61dCWQYOd6CTxl4hjya9dsfEZGsM4uGOUdax60lEI4twsXGXf7crmz8Gk__GhupTrWxA7RFRVt76',
+            'info_dict': {
+                'id': '2431867_001',
+                'title': 'こながめでたい日２０２４の視聴ページ | PIA LIVE STREAM(ぴあライブストリーム)',
+                'live_status': 'was_live',
+            },
+            'params': {
+                'skip_download': True,
+                'ignore_no_formats_error': True,
+            },
+        },
+        {
+            'url': 'https://player.pia-live.jp/stream/4JagFBEIM14s_hK9aXHKf3k3F3bY5eoHFQxu68TC6krJdu0GVBVbVy01IwpJ6J3qBEm3d9TCTt1d0eWpsZGj7DrOjVOmS7GAWGwyscMgiThopJvzgWC4H5b-7XQjAfRZ',
+            'info_dict': {
+                'id': '2431867_002',
+                'title': 'こながめでたい日２０２４の視聴ページ | PIA LIVE STREAM(ぴあライブストリーム)',
+                'live_status': 'was_live',
+            },
+            'params': {
+                'skip_download': True,
+                'ignore_no_formats_error': True,
+            },
+        },
+    ]
+
+    def handle_embed_player(self, player_tag, video_id, info_dict={}):
+        player_data_url = self._search_regex([PIAULIZAPortalAPIIE.TAG_REGEX_PATTERN],
+                                             player_tag, 'player data url', fatal=False)
```

Comment:
Why `[pattern]` ?

Reply:
Since I want to store this regular expression in a constant, maybe calling it `TAG_REGEX` would be more appropriate?